charts/astronomer/values.yaml

```diff
@@ -205,6 +205,14 @@ houston:
     # Default here is to run at 00:43 every night https://crontab.guru/#43_0_*_*_*
     schedule: "43 0 * * *"
 
+  # Populate daily task usage data
+  # This runs as a CronJob
+  populateHourlyTaskAuditMetrics:
+    # Enable check updates CronJob
+    enabled: true
```

Comment:
we can remove this altogether and enable this cron whenever taskUsageMetrics feature flag is enabled

Reply:
if i remove the block then have to put schedule somewhere else